jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenter.java

```diff
@@ -320,9 +320,9 @@ public void setupAdvancedSearchView() {
 
         final Map<String, String> acks = new HashMap<>();
         final org.jbpm.workbench.common.client.resources.i18n.Constants constants = org.jbpm.workbench.common.client.resources.i18n.Constants.INSTANCE;
-        acks.put(String.valueOf(Boolean.TRUE),
+        acks.put(String.valueOf(new Short("1")),
```

Comment:
@cristianonicolai But `addSelectFilter()` expects a `Map<String, String> options`. I guess we could directly `put` `"1"` and `"0"` instead of converting from String to Short and then back to String. I just used the `new Short(String)` to follow Maciej's code convention. Wdyt?